adminSDK/directory/index.html

```diff
@@ -57,7 +57,15 @@
        * Callback after api.js is loaded.
        */
       function gapiLoaded() {
+      try
+      {
         gapi.load('client', intializeGapiClient);
+        }
+      catch(ex)
```

Comment:
Formatting still seems off.

Also, not sure if this try/catch is needed here. Don't think gapi.load is expected to throw anything, though could be wrong here. Under what conditions is an exception expected?